### ISSUE TITLE
Installer checks not for hyper v and container feature

### ISSUE THREAD

**Issue by StefanSchoof** [Azure/iotedge#1068]
## Expected Behavior
Start a installation for a windows device with the guide from https://docs.microsoft.com/en-us/azure/iot-edge/how-to-install-iot-edge-windows#install-iot-edge-on-a-new-device
If the Windows has not activated hyper V and Container Feature, the installer reports that this is required and iotedge wont work without this

## Current Behavior
Without hyper v the installation completes, but the iotedge service cloud not start. There is no good error message that point that there is an requirement missing. With hyper v, but without the container features iotedge starts and downloads the edgeAgent image, but is not able to start. In iotedge-moby log I found.

```
/containers/edgeAgent/restart returned error: Cannot restart container edgeAgent:
                    hcsshim::CreateComputeSystem 4c448d366cd1d80f846d51fdbc65573f46a73e23fb3c485eea5c91a7fda1e1bb: Die
                    Anforderung wird nicht unterstützt.
```
This also not indicate a missing requirement

## Steps to Reproduce

1. Install a new Windows 10 (in my case a Pro)
2. Run `. {Invoke-WebRequest -useb aka.ms/iotedge-win} | Invoke-Expression; 
Install-SecurityDaemon -Manual -ContainerOs Windows -DeviceConnectionString '<connection-string>'`
3. Service is not starting
4. Run `Enable-WindowsOptionalFeature -Online -FeatureName Microsoft-Hyper-V -All`
5. Crush Head on Keyboard, Contacting Microsoft Support, escalate installation problems ...
5. Service is starting but container start fails
6. Run `Enable-WindowsOptionalFeature -Online -FeatureName containers –All`
7. All Containers are starting and working

### Device (Host) Operating System
Windows 10

### Architecture
amd64

### Container Operating System
Windows containers

#### iotedged
iotedge 1.0.6.1 (3fa6cbef8b7fc3c55a49a622735eb1021b8a5963)

#### Edge Agent
1.0

#### Edge Hub
1.0

#### Docker
iotedge-moby

## Additional Information

Support Case Number: 119040922001123

**Comment by veyalla** [comment]
This will be fixed in 1.0.7 release.

**Comment by myagley** [comment]
This has been released as part of the 1.0.7 release candidate. You can get packages here: https://github.com/Azure/azure-iotedge/releases/tag/1.0.7-rc2
